activation/activation.go

```diff
@@ -222,6 +235,53 @@ func (b *Builder) buildNipstChallenge(epoch types.EpochId) error {
 	return nil
 }
 
+func (b *Builder) StartPost(rewardAddress address.Address, dataDir string, space uint64) error {
+	b.log.Info("Starting post, reward address: %x", rewardAddress)
+	b.SetCoinbaseAccount(rewardAddress)
+	b.postInitLock.Lock()
+	if b.initStatus != Idle {
```

Comment:
Last time I wrote:

> It might be worth to check for `InProgress` status directly, instead of assuming that's the case if we're not on `Idle` && `IsPostInitialized()` returns false. Whatever you think.

Now after `IsPostInitialized()` removal, you're not covering the `Done` case.
So this need to be fixed, and also you should set the `Done` statue value if you're getting "already initialized" err from the initializer. Also, IMO "already initialized" should not be appended to "PoST initialization failed: " string.

Reply:
Why not to do this now as `if b.initStatus == InProgress`?